### DISCUSSION TITLE
Revset to find branches being worked on by a specific person

### DISCUSSION THREAD

**Discussion by JaniM** [jj-vcs/jj#3383] [Show and tell]
This is the only revset alias I keep around as it's massively useful. It finds every commit in unmerged branches that have at least one commit from the given person. Handy when a coworker asks you to check their work out.

    "work(x)" = "immutable_heads()..((immutable_heads().. & (author(x) | committer(x)))::)"

**Comment by amiryal** [comment]
Thanks for sharing! I think it can be simplified as:
```
"work(x)" = "(immutable_heads()..(author(x) | committer(x)))::"
